lib/src/golden_test.dart

```diff
@@ -132,50 +196,60 @@ Future<void> goldenTest(
   PumpWidget pumpWidget = onlyPumpWidget,
   Interaction? whilePerforming,
   required ValueGetter<Widget> builder,
-}) async {
-  if (skip) return;
+}) =>
+    _runGoldenTest(
+      description,
+      getImageFn: goldenTestAdapter.getImage,
```

Comment:
just out of curiosity, can you explain what the getImage functions are doing and why they're passed in `_runGoldenTest`?

Reply:
They are capturing the actual image to compare. This way, the animation test is run in almost the same way as normal ones, but when capturing the image we capture every frame rather than a single snapshot.